content/WebAuthn/Concepts/Authenticator_Management/Use_Cases_and_Scenarios.adoc

```diff
@@ -0,0 +1,117 @@
+
+=== Limit authenticators based on device characteristics/features
+
+This scenario is where you begin to add controls to make your application dynamically respond to different authenticators. This will allow you to add a higher degree of security to user accounts, while not constraining account holders to specific authenticators. Understanding this section will be critical for implementing link:https://developers.yubico.com/Developer_Program/WebAuthn_Starter_Kit/Adaptive_Multi-Factor_Authentication.html[adaptive multi factor authentication] into your application; your relying party will be able to adapt and route users across different flows based on the risk profile of the authenticator used, and the action being attempted by the user. 
+
+It’s worth noting that this category is where you will begin to introduce friction for users who are not leveraging high assurance devices. A user who is leveraging a high assurance device will be able to seamlessly authenticate due to the very high degree of confidence that their request is legitimate. Authentication ceremonies performed with lower assurance devices may require another factor to authenticate, or step-up authentication to perform certain actions. 
+
+This scenario works best when:
+
+* A user's authenticator is missing user verification
+* A user is attempting to use a copyable passkey in a sensitive environment
+
+==== Implementation guidance
+
+Below is implementation guidance for two separate user scenarios: adding an additional factor to a WebAuthn authentication ceremony, and performing step-up authentication for low assurance devices.
+
+link:/WebAuthn/Concepts/Authenticator_Management/Implementation_Guidance/Adding_Additional_Authentication_Factors.html[Adding an additional authentication factor implementation guidance]
+
+link:/WebAuthn/Concepts/Authenticator_Management/Implementation_Guidance/Step_Up_Authentication.html[Step up authentication implementation guidance]
+
+=== Deny specific FIDO2 authenticators
+
+In the previous sections we were highly permissive in the authenticators allowed in our environment, and even if a device wasn’t completely trusted we supplied options to allow them to be used. As we transition into this section we will begin to entirely block certain authenticators from an environment. This means that any attempt to register one of these authenticators in our application will be rejected, forcing the user to use a valid authenticator before proceeding. 
+
+Within your environment you will create a curated list of authenticators that will be rejected by your application. This list could be based off of an external repository such as the FIDO MDS that tracks authenticators with known issues, or a manually curated list of authenticators deemed untrustworthy by your organization. 
+
+This scenario works best when:
+
+* Your organization has a security organization actively monitoring for authenticator issues
```

Comment:
Consider replacing the first instance of organization with 'company' or 'enterprise'.